kml/test_waypoints_to_kml.py

```diff
@@ -17,10 +18,15 @@ def waypoints():
     """
     Waypoints input.
     """
-    return [(37.7749, -122.4194), (34.0522, -118.2437), (40.7128, -74.0060)]
 
+    return [
+        location_ground.LocationGround(0, 37.7749, -122.4194), 
+        location_ground.LocationGround(1, 34.0522, -118.2437), 
+        location_ground.LocationGround(2, 40.7128, -74.0060)
```

Comment:
The constructor expects a string, but an integer is being passed instead. Also, change the names to someting more descriptive (e.g. San Francisco, Los Angeles, New York City as those are the corresponding latitude longitude coordinates).